### PR TITLE
Update node dependencies & add Apache LICENSE/preambles

### PR DESCRIPTION
**Pull request by wrboyce**
* update node dependencies
* add Apache 2.0 LICENSE and preambles

Change-type: major
Signed-off-by: Will Boyce \<will@resin.io\>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,32 +5,33 @@
...
     "request": "^2.65.0"
   },
   "devDependencies": {
     "atob": "^2.0.0",
-    "chai": "^3.0.0",
-    "chai-as-promised": "^5.1.0",
-    "coffee-script": "~1.10.0",
+    "chai": "^4.2.0",
+    "chai-as-promised": "^7.1.1",
+    "coffeescript": "^2.3.2",
```
Comment (Page-) [inline package.json:27]: This is potentially a breaking change as I know coffeescript 2 changes the target es version for compilation compared to coffeescript 1

File: package.json [hunk 2]
```diff
@@ -5,32 +5,33 @@
...
     "pretest": "npm run lint && coffee -c index.coffee",
-    "test": "mocha --compilers coffee:coffee-script/register test/index.coffee",
+    "test": "mocha --require coffeescript/register test/index.coffee",
     "prepublish": "npm test"
   },
   "dependencies": {
     "bluebird": "^3.0.0",
-    "jsonwebtoken": "^5.0.4",
-    "passport-jwt": "^2.1.0",
+    "jsonwebtoken": "^8.3.0",
```
Comment (Page-) [inline package.json:19]: `createJwt` directly passes its args through to jsonwebtoken, can you check that none of the args it passes are affected by the breaking changes in jsonwebtoken

Comment (wrboyce) [reply package.json:19]: Yeah, there don't appear to be any changes that affect us.

File: package.json (hunk 1 above)
Comment (wrboyce) [reply package.json:27]: fixed

**Review by Page- — APPROVED** [review]

**Pull request merged**